azure-pipelines-ui-tests.dfe.yml

```diff
@@ -224,3 +225,212 @@ jobs:
     inputs:
       path: tests/robot-tests/test-results/
       artifactName: test-results-admin-public
+
+
+- job: 'PublicPlaywrightUItest'
+  displayName: Public suite - Playwright UI tests
+  timeoutInMinutes: 160
+  cancelTimeoutInMinutes: 10
+  condition: succeededOrFailed()
+  pool: ees-ubuntu2204-large
+  workspace:
+    clean: all
+  steps:
+  - checkout: self
+    clean: true
+    fetchDepth: 5
+    fetchTags: false
+
+  - task: NodeTool@0
+    displayName: Use Node version 18 above
+    timeoutInMinutes: 5
+    inputs:
+      versionSpec: "18.18.2"
+
+  - script: |
```

Comment:
Does this task need to remain or was it just used during debugging?  If it stays, I'd probably just clean up its display name to something like the others, like "Display Node version".